extension/source/App.ts

```diff
@@ -192,7 +190,7 @@ export default class App {
       return undefined;
     }
 
-    return this.blsWalletSigner.getPublicKey(this.state.privateKey);
+    return this.wallet?.blsWalletSigner.getPublicKeyHash(this.state.privateKey);
```

Comment:
Should be `getPublicKey` right? Not `getPublicKeyHash`?

(If the public key hash is what we really want then we should rename the method.)

Reply:
I added this as typing filler, `getPublicKey` now returns an `Array(4)` instead of a string. This will be changed in future commits.

Reply:
Added `getPublicKeyStr` to signer and `PublicKey` and `PublicKeyStr` to `BLSWalletWrapper`.

Reply:
Interesting. I want to aim for handling these things in a more uniform way but not sure whether it's suitable here or not. In any case, it's something I'd rather fix as a followup than have in this PR if it looks like a good idea later.

Reply:
("Handling in a uniform way" means using the string tuple everywhere instead of converting between formats for different projects.)